### PR TITLE
Maboelsoudd2l/toggle different sizes

### PR DESCRIPTION
**Pull request by maboelsoudd2l**
Checked rtl and different browsers (chrome, safari, firefox, osx-edge).
This is what it looks like in different sizes:
Screen Shot 2019-08-06 at 2 47 35 PM
Screen Shot 2019-08-06 at 2 47 16 PM
Screen Shot 2019-08-06 at 2 46 56 PM

### CODE REVIEW

File: components/d2l-quick-eval/d2l-quick-eval-view-toggle.js [hunk 1]
```diff
@@ -12,6 +12,12 @@ class D2LQuickEvalViewToggle extends QuickEvalLocalize(PolymerElement) {
 	static get template() {
 		const toggleTemplate = html`
 			<style>
+				:host {
+					margin: 0 -0.9rem;
+				}
+				label {
+					margin: 0 0.9rem;
```
Comment (maboelsoudd2l) [inline components/d2l-quick-eval/d2l-quick-eval-view-toggle.js]: A fix for https://rally1.rallydev.com/#/detail/userstory/323678156208?fdp=true

File: components/d2l-quick-eval/d2l-quick-eval-view-toggle.js [hunk 2]
```diff
@@ -12,6 +12,12 @@ class D2LQuickEvalViewToggle extends QuickEvalLocalize(PolymerElement) {
 	static get template() {
 		const toggleTemplate = html`
 			<style>
+				:host {
+					margin: 0 -0.9rem;
```
Comment (maboelsoudd2l) [inline components/d2l-quick-eval/d2l-quick-eval-view-toggle.js]: Magical css inspired by bootstrap.

File: components/d2l-quick-eval/d2l-quick-eval-activities.js [hunk 3]
```diff
@@ -85,6 +85,26 @@ class D2LQuickEvalActivities extends mixinBehaviors(
 				.d2l-body-standard {
 					@apply --d2l-body-compact-text;
 				}
+				@media (min-width: 525px) {
+					.d2l-quick-eval-activity-list-modifiers {
+						width: auto;
+						float: left;
```
Comment (AlexBedley) [inline components/d2l-quick-eval/d2l-quick-eval-activities.js]: Does this change support RTL?

Comment (AlexBedley) [reply components/d2l-quick-eval/d2l-quick-eval-activities.js]: oh I see the rtl selectors. Looks good

**Comment by AlexBedley** [comment]
In the medium size (when the viewby and filter/search are left aligned) it looks a bit weird because they are offset a little. Screenshot: 
image


I'd expect their left edge to match (right now the filter/search is indented more). What do the designers think?

**Comment by AlexBedley** [comment]
Another thing I found - in medium mode the filter panel opens above the button, whereas in big and small mode it opens below. I think we want the filter panel to always open below. On the demo page the filter panel is super small because it opens above which makes it less usable.

image

**Comment by AlexBedley** [comment]
I tested this on windows edge and it looks good. Nice!

**Comment by maboelsoudd2l** [comment]
> In the medium size (when the viewby and filter/search are left aligned) it looks a bit weird because they are offset a little. Screenshot:
> image
> 
> I'd expect their left edge to match (right now the filter/search is indented more). What do the designers think?

Oops, completely forgot to update the filter in Activities. Just force-pushed the changes to fix this

**Comment by maboelsoudd2l** [comment]
> Another thing I found - in medium mode the filter panel opens above the button, whereas in big and small mode it opens below. I think we want the filter panel to always open below. On the demo page the filter panel is super small because it opens above which makes it less usable.
> 
> image

So this is not something in quick-eval, the dropdown is coded up to be below if there is sufficient space. If there is no space, the dropdown is above. https://github.com/BrightspaceUI/dropdown/blob/f28df5196e10fabc05cec4b33bca324b4b5c9ebf/d2l-dropdown-content-behavior.js#L630 I assume that behaviour makes sense.

I found that if you scroll down so that space below > space above, the dropdown automatically moves down as expected.

**Comment by AlexBedley** [comment]
I verified that your push fixes the indenting issue. Nice.

I'm cool with the dropdown up/down thing.

**Pull request merged**